app/jobs/transfer_basespace_files.rb

```diff
@@ -18,15 +18,17 @@ def self.perform(sample_id, basespace_dataset_id, basespace_access_token)
     if samples_remaining.empty?
       begin
         BasespaceHelper.revoke_access_token(basespace_access_token)
-      rescue
+      rescue HttpHelper::HttpError => e
         # If revoke_access_token is called multiple times, it will fail on the subsequent calls. This is okay.
-        nil
+        if e.status_code == 401
+          Rails.logger.warn("Revoke access token failed. Likely called multiple times by samples sharing the same token.")
```

Comment:
Should we add the sample id here? (ex: "Revoke access token failed for sample 123. Likely...")